tracer/test/test-applications/regression/AspNetCoreSmokeTest/AspNetCoreSmokeTest.csproj

```diff
@@ -1,12 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0;net8.0;net9.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net462;$(TargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0;net8.0;net9.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(PublishTrimmed)' != 'true'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;$(TargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(PublishTrimmed)' != 'true' AND '$(OS)' == 'Windows_NT'">net462;$(TargetFrameworks)</TargetFrameworks>
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(PublishTrimmed)' == 'true'">
+    <PublishTrimmed>true</PublishTrimmed>
+    <SelfContained>true</SelfContained>
+    <PublishSingleFile>true</PublishSingleFile>
+    <NoWarn>IL2007;IL2008;IL2026;IL2104</NoWarn>
+    <DefineConstants>$(DefineConstants);PUBLISH_TRIMMED</DefineConstants>
+    <PublishSingleFile>true</PublishSingleFile>
```

Comment:
`PublishSingleFile` is already set a few lines above.

Reply:
Oops, thanks 👍

Reply:
```suggestion
```